aspnet/web-forms/overview/older-versions-getting-started/deploying-web-site-projects/configuring-the-production-web-application-to-use-the-production-database-cs.md

```diff
@@ -54,11 +54,11 @@ Next, specify the various database connection information (see Figure 2). When y
 
 **Figure 2**: Specify the Database Connection Information ([Click to view full-size image](configuring-the-production-web-application-to-use-the-production-database-cs/_static/image6.jpg))
```

Comment:
https://docs.microsoft.com/en-us/aspnet/web-forms/overview/older-versions-getting-started/deploying-web-site-projects/configuring-the-production-web-application-to-use-the-production-database-cs has too many passwords to salvage. I think we should delete this file and redirect to one of the ASP.NET Core/SQL Server to Azure tutorials.

Reply:
Get the traffic on this. I don't think many folks are reading the entire doc.

Reply:
I sent the data by email to you.  You are correct, the few hitting the doc are bouncing out.

Reply:
will delete.